lib/DDG/Spice/HealthScores.pm

```diff
@@ -0,0 +1,16 @@
+package DDG::Spice::HealthScores;
+# ABSTRACT: Returns restaurant health score information from the 
+#           Eaternet/Swiftype API.
+
+use strict;
+use DDG::Spice;
+
```

Comment:
Can you fill in metadata here https://duck.co/duckduckhack/metadata 

+ description
+ primary_example_queries
+ category
+ topics
+ code_url
+ attribution